src/Components/Steps/Steps.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from "react";
+import { observer } from "mobx-react";
+import styles from "./Steps.module.css";
+import type { MediaPresenter } from "../../MediaPresenter";
+import type { OutputPresenter } from "../../OutputPresenter";
+import { showError } from "../ErrorToast/ErrorToast";
+
+type Props = {
+  mediaPresenter: MediaPresenter;
+  outputPresenter: OutputPresenter;
+  openPlayerModal: () => void;
+  closePlayerModal: () => void;
+};
+
+export const Steps: React.FC<Props> = observer(
+  ({ mediaPresenter, outputPresenter, openPlayerModal, closePlayerModal }) => {
+    const handleMediaUpload = async (
+      event: React.ChangeEvent<HTMLInputElement>
+    ) => {
+      if (event.target.files === null) return;
+      const files: FileList = event.target.files;
+      Array.from(files).forEach((file: File) => {
+        if (!mediaPresenter.addFile(file)) {
+          showError(file.name + "is not an acceptable file format");
+        }
+      });
+    };
+
+    const [stepOneOpen, setStepOneOpen] = useState<boolean>(true);
+    const [stepTwoOpen, setStepTwoOpen] = useState<boolean>(false);
```

Comment:
Looks like you're the modal step state in here but also calling `openPlayerModal` and `closePlayerModal` 🤔 
Consider moving the open state out of this  component and in to a presenter that can then expose the current step and functions to open/close steps.